### ISSUE TITLE
Unable to manage default hostname for a pullzone

### ISSUE THREAD

**Issue by spaceemotion** [BunnyWay/terraform-provider-bunnynet#12]
We have the following resource:

```tf
resource "bunnynet_pullzone_hostname" "this" {
  pullzone    = bunnynet_pullzone.this.id
  name        = "${bunnynet_pullzone.this.name}.${bunnynet_pullzone.this.cdn_domain}"
  tls_enabled = true
  force_ssl   = true
}
```

We added it, because we want SSL to be enabled (which isn't the default case), but it fails, as the hostname already exists (since the pullzone creation itself seems to have added one already).

Is there a way to manage the default hostname? The pullzone resource does not seem to have the additional options it seems.

**Comment by spaceemotion** [comment]
Oh yeah, i know how to import. The issue was that we want to have the options set up when we create a new pullzone. There doesn't seem to be a way to do it in one swoop?

here's what we did
1. We have a module for the BunnyCDN stuff
2. It created the pullzone
3. It failed at the hostname, cause the pullzone already created it
4. We had to import the hostname
5. The next `terraform apply` now set the SSL options correctly.
